src/schema-gen/InspectorXSLT/testing/OSCAL-INSPECTOR-XSLT.xpl

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<p:declare-step xmlns:p="http://www.w3.org/ns/xproc" xmlns:c="http://www.w3.org/ns/xproc-step" version="1.0"
+   xmlns:metaschema="http://csrc.nist.gov/ns/metaschema/1.0" type="metaschema:OSCAL-INSPECTOR-XSLT"
+   name="OSCAL-INSPECTOR-XSLT">
+
+   <!-- Purpose: Produces an Inspector XSLT for the OSCAL Catalog metaschema  -->
+   <!-- Input: is hard-wired to a current copy of the metaschema on Github -->
+   <!-- Output: Writes artifacts to the file system, in 'current' subdirectory -->
```

Comment:
Minor, non-blocking: it seems important to say why it is `_RAW` to me, since it impacts downstream processing later.

```suggestion
   <!-- Output: Writes artifacts to the file system, in 'current' subdirectory, encoding the source document into "raw" ASCII with escaped characters, if applicable.  -->
```